packages/components/src/modal/frame.js

```diff
@@ -28,15 +28,27 @@ class ModalFrame extends Component {
 		this.handleKeyDown = this.handleKeyDown.bind( this );
 		this.handleFocusOutside = this.handleFocusOutside.bind( this );
 		this.focusFirstTabbable = this.focusFirstTabbable.bind( this );
+		this.focusOnMount = this.focusOnMount.bind( this );
 	}
 
 	/**
-	 * Focuses the first tabbable element when props.focusOnMount is true.
+	 * Focuses the container or first tabbable element based on props.focusOnMount.
 	 */
 	componentDidMount() {
 		// Focus on mount
 		if ( this.props.focusOnMount ) {
+			this.focusOnMount();
+		}
+	}
+
+	/**
+	 * Handle focus on mount.
+	 */
+	focusOnMount() {
+		if ( this.props.focusOnMount === 'firstElement' ) {
 			this.focusFirstTabbable();
+		} else {
+			this.containerRef.current.focus();
```

Comment:
@fernandovbs Hai 👋 !

Thank you for working on this :). I took an initial look. Pulled it down and tested locally.
This `else` would qualify `focusOnMount="container"`. Reading it though, it took me a bit to understand it.

What are your thoughts on clarifying the condition a bit. Maybe something like:

```
if (this.props.focusOnMount === 'container' || this.props.focusOnMount === true)
```

It's slightly more verbose, but I feel like it's clearer.
Alternatively, we could use a `switch` statement for this logic flow

Would love your thoughts!

Reply:
Hi @ItsJonQ !
I think the first option is great :). 
I'll update my PR and also include the stories update into this one.
Thank you for taking the time to review it.